packages/@webex/webex-core/src/lib/services-v2/service-registry.js

```
@@ -0,0 +1,465 @@
+      active: true,
+      local: true,
+      priority: true,
+    }).reduce((map, host) => {
+      // Generate a new object to assign the existing map.
+      const hostReference = {};
+
+      // Assign the key:value pair for the service and url.
+      hostReference[host.service] = host.url;
+
+      // Assign the reference to the map and return.
+      return {...map, ...hostReference};
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)